xclim/testing/tests/test_indices.py

```diff
@@ -1045,6 +1045,41 @@ def test_below_simple(self, tasmax_series):
         np.testing.assert_array_equal(out[1:], [0])
 
 
+class TestJetStreamIndices:
+    # data needs to consist of at least 61 days for Lanczos filter (here: 66 days)
+    time_coords = pd.date_range("2000-01-01", "2000-03-06", freq="D")
+    # make random ua data array of shape (66 days, 3 plevs, 5 lons, 5 lats)
+    np.random.seed(42)
+    da_ua = xr.DataArray(
+        np.random.rand(66, 3, 5, 5),
+        coords={
+            "time": time_coords,
+            "plev": [75000, 85000, 100000],
+            "lon": [-60, -59, -58, -57, -56],
+            "lat": [15, 16, 17, 18, 19],
+        },
+        dims=["time", "plev", "lon", "lat"],
+        attrs={
+            "standard_name": "eastward_wind",
+            "units": "m s-1",
+        },
+    )
+    da_ua.plev.attrs["units"] = "Pa"
+
+    def test_jetstream_metric_woolings(self):
+        da_ua = self.da_ua
+        out = xci.jetstream_metric_woolings(da_ua)
+        np.testing.assert_equal(len(out), 2)
+        jetlat, jetstr = out
+        # should be 6 values that are not NaN because of 61 day moving window and 66 chosen
+        np.testing.assert_equal(np.sum(~np.isnan(jetlat).data), 6)
+        np.testing.assert_equal(np.sum(~np.isnan(jetstr).data), 6)
+        np.testing.assert_equal(jetlat.max().data, 19.0)
+        np.testing.assert_equal(jetstr.max().data, 0.5620588628647811)
```

Comment:
How were the expected values determined ? Or in other words, what reference are we checking against to make sure it works as expected ?

Reply:
I haven't yet been able to solve this comment yet, so any advice would be welcome! Currently, I am taking a random array of numbers (random seed==42) of shape: 66 days, 3 plev, 5 lat, 5 lon, as this function needs at minimum of 61 days for the Lanczos filter and then I have checked the data before and results manually. One way of getting a reference to check this function might be using actual ua data (maybe the dataset used in Woolings et al. 2010 paper), but then the test might take too long to run. Still unsure...

Reply:
Then just add a comment stating that you've computed the expected answer manually. 

Would it be possible to create a very simple synthetic data set (e.g `np.ones()`) where the answer can be computed easily?

Reply:
Okay, I have made changes and added an improved test with np.ones/np.zeros test in: ff3d1dd1c7cda53dc14b2e14a85eb076c1723b53. Hope this is okay :)